README.md

```diff
@@ -4,7 +4,7 @@
 https://github.com/ibm-cloud-architecture/refarch-cloudnative*
 
 
-The BlueCompute app consists of an iOS and Android version. The iOS application is developed in Swift 2.0. The mobile apps were built to demonstrate the following capability on IBM Cloud:
+The BlueCompute app consists of an iOS and Android version. The iOS application is developed in Swift 3.0 with support for older versions. The mobile apps were built to demonstrate the following capability on IBM Cloud:
```

Comment:
Specifying this as some of the dependencies have not been updated to Swift 3 yet